docker/gofer/client.json

```diff
@@ -0,0 +1,3 @@
+{"ethereum": {
+  "rpc": "http://geth.local:8545"
```

Comment:
I am not fond of the idea that a thin client needs that. 
I propose to create a new binary with only the client - e.g. `gofer` & `goferd`